src/frontend/components/Felleskomponenter/Steg/useFormProgressSteg.tsx

```diff
@@ -0,0 +1,100 @@
+import { useApp } from '../../../context/AppContext';
+import { useSteg } from '../../../context/StegContext';
+import { ISteg, RouteEnum } from '../../../typer/routes';
+import { FlettefeltVerdier, LocaleRecordBlock } from '../../../typer/sanity/sanity';
+
+interface IStegMedTittel extends ISteg {
+    tittel: string;
+}
+
+export const useFormProgressSteg = (): IStegMedTittel[] => {
+    const { tekster, plainTekst } = useApp();
+    const { steg, barnForSteg } = useSteg();
+
+    const {
+        FORSIDE,
+        OM_DEG,
+        DIN_LIVSSITUASJON,
+        VELG_BARN,
+        OM_BARNA,
+        OM_BARNET,
+        OPPSUMMERING,
+        DOKUMENTASJON,
+        EØS_FOR_BARN,
+        EØS_FOR_SØKER,
+        KVITTERING,
+    } = tekster();
+
+    const antallBarnTellere = {
```

Comment:
Lurer på om det er like greit å bare ha 2 variabler her

Reply:
Enig, tenker det enkleste og mest lesbare her er å ha to konstanter i stedet for et objekt med to key value pairs.